app/views/fae/application/_header.slim

```diff
@@ -3,10 +3,10 @@ header.main-header#js-main-header
   h1: a href=root_path = @option.title
 
   - if @fae_topnav_items.present?
-    nav.js-nav
-      ul.main-header-nav
+    nav
+      ul.main-header-nav.js-nav data-coordinates=@fae_navigation.coordinates[0..1].join(',')
```

Comment:
It made the Slim look super gross, and the JS solution was less lines. In the Slim solution, I found I was writing the same code over and over again, but it was too small (ternary here, interpolation there) to modularize. 

If you prefer this to be a Slim solution, I can go back to that.